<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-G4P6S3Q" ignored="0" name="Default test(failed)" tests="1" failures="1" timestamp="2023-09-09T12:19:46 IST" time="83.822" errors="0">
  <testcase name="testInvalidSignup" time="15.872" classname="TestCases.TC_01_SignUp">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestCases.TC_01_SignUp.testInvalidSignup(TC_01_SignUp.java:136)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testInvalidSignup -->
</testsuite> <!-- Default test(failed) -->
